### ISSUE TITLE
Not properly handling max_sustained_wind_unit in TropCyclone.from_tracks

### ISSUE THREAD

**Issue by ThomasRoosli** [CLIMADA-project/climada_python#694]
Some windfield models in the `TropCyclone `class use `max_sustained_wind` as input. They expect the unit to be in knots and produce very different results otherwise. The below code shows that the max intensity of the resulting hazard changes in a certain setup changes up to 40% in a certain setup. We should make the function `from_tracks` be aware of the field `max_sustained_wind_unit` in `TCTracks`.

code to replicate the issue:

```
"""Value to test against """
intensity_values = {
    "ER11": 45.60641725103528,
}


"""Test _tc_from_track function with knots."""
tc_track = TCTracks.from_processed_ibtracs_csv(TEST_TRACK)
tc_track.equal_timestep()
tc_track.data = tc_track.data[:1]

for model in ["ER11"]:
    tc_haz = TropCyclone.from_tracks(tc_track, centroids=CENTR_TEST_BRB, model=model)
    assert tc_haz.units  == 'm/s'
    np.testing.assert_almost_equal(tc_haz.intensity.max(), intensity_values[model])
    


"""Test _tc_from_track function with different wind meters/second."""
tc_track = TCTracks.from_processed_ibtracs_csv(TEST_TRACK)
tc_track.equal_timestep()
tc_track.data = tc_track.data[:1]
#adjust data to m/s from kn
for data_i in tc_track.data:
    if data_i.max_sustained_wind_unit == 'kn':
        data_i['max_sustained_wind'] = xr.DataArray(
            (
                data_i.max_sustained_wind.values * ureg.knots
                ).to(ureg.meter/ureg.second).magnitude,
            coords = data_i['max_sustained_wind'].coords
            )            
        data_i.attrs['max_sustained_wind_unit'] = 'm/s'

for model in ["ER11"]:
    tc_haz = TropCyclone.from_tracks(tc_track, centroids=CENTR_TEST_BRB, model=model)
    assert tc_haz.units  == 'm/s'
    np.testing.assert_almost_equal(tc_haz.intensity.max(), intensity_values[model])  #raises AssertionError
```
The same problem exists with the Holland 1980 model.

**Comment by bguillod** [comment]
Thanks for highlighting this issue.
Another important aspect to consider here: some agencies report `max_sustained_wind` as 1-minute averages (US agencies) and other as 10-minutes averages (e.g. BOM) or even different.
I suggest that an additional attribute is included in `TCtracks` objects which state the averaging time of this variable, and that this attribute is also taken into account in these algorithms by first rescaling values to the averaging time required by the respective wind models.

**Comment by ThomasRoosli** [comment]
I agree @bguillod and opened a seperate issue for the averaging period #748

**Comment by ThomasRoosli** [comment]
The issue was resolved in #749 using some suggested changes from the closed pull request #747. `TropCyclone.from_tracks` is now able to correctly handle different `max_sustained_wind_unit`. This issue can be closed with the next release.

**Comment by peanutfun** [comment]
Thanks for the feedback. Closing
